collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,291 @@
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/dev-snapshot-godot-4-4-dev-4/physics-placement.mp4?1" type="video/mp4">
+</video>
+
+To reduce clutter in your inspector and highlight the properties you care the most about, YeldhamDev brings us the long-awaited ability to pin one’s favorite properties in the inspector! Check out the implementation in PR [GH-97352](https://github.com/godotengine/godot/pull/97352).
+
+Thanks to Danil Alexeev the GDScript code editor will now display a tooltip containing information about functions, variables, classes, etc. when you hover over them, including the documentation you have written using our documentation system ([GH-91060](https://github.com/godotengine/godot/pull/91060)). This makes using the integrated documentation system even more powerful as you no longer have to bounce between the code editor and the related docs to quickly get information.
+
+### GDScript
+This release GDScript saw quite a few quality of life features to improve your development experience. For example, you can now create buttons in the inspector seemlessly using the new `@export_tool_button` annotation in `@tool` scripts. Added by Macksaur in [GH-96290](https://github.com/godotengine/godot/pull/96290).
+
+Danil Alexeev brings us `@warning_ignore_start` and `@warning_ignore_restore` ([GH-76020](https://github.com/godotengine/godot/pull/76020)) to suppress warnings in a safely scoped manner across entire sections of code. Similarly, `@warning_ignore` no longer works when applied to a function. This is because it previously had undocumented functionality which applied the warning to the *entire function* unexpectedly. Now users have much more control and the annotations should function more like what you would expect.
+
+### Import
+4.4 is coming with huge improvements to texture importing quality and speed thanks to new maintainer Bluecube3310. The import times of textures using the "VRAM Compressed" import setting have been greatly improved by integrating the GPU-based Betsy texture compressor. The biggest difference is observable when importing HDR images (such as HDRIs or lightmaps), which used to take up to several minutes before. Thanks to Betsy, as well as other optimizations to the import process ([GH-92291](https://github.com/godotengine/godot/pull/92291) and [GH-95291](https://github.com/godotengine/godot/pull/95291)), that time has gone down significantly. Additionally, textures not using the "High Quality" import setting should look better by default, since previously we had to trade the quality for faster import times. Additionally, there have been countless improvements to the texture import process to improve compatibility across devices and the number of texture formats supported. Overall, texture importing should feel faster and more stable than ever before.
```

Comment:
Two "additionally" in a row is a bit repetitive, maybe "furthermore" or something